final/Makefile

```diff
@@ -1,27 +1,41 @@
+.PHONY: smooth convolve glm normality conclusion analysis figs all clean 
+
+# FIX INDIVIDUAL COMMANDS
 smooth:
-	python scripts/smoothing.py
+	cd scripts && python smoothing.py
 
 convolve:
-	python scripts/convolution.py
+	cd scripts && python convolution.py
 	
 glm:
-	python scripts/glm.py
+	cd scripts && python glm.py
 
 normality:
-	python scripts/normality.py
+	cd scripts && python py
 
 conclusion:
-	python scripts/conclusion.py
-
-all:
-	python scripts/smoothing.py
-
-	python scripts/convolution.py
-	
-	python scripts/glm.py
-
-	python scripts/normality.py
-
-	python scripts/conclusion.py
-
-clean:
\ No newline at end of file
+	cd scripts && python conclusion.py
+
+# FIX
```

Comment:
We'll look at this today, and probably have someone run all the makefiles that we think are good